Xamarin.Essentials/Types/StorageExtensions.shared.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Xamarin.Essentials;
+
+namespace Xamarin.Essentials.Types
+{
+    public static class StorageExtensions
+    {
+        public static byte FreePercentage(this StorageInfo info) =>
+            Convert.ToByte(Math.Round(FreePercentage(info.FreeBytes, info.Capacity) * 100));
+
+        public static byte UsedPercentage(this StorageInfo info) => Convert.ToByte(Math.Round(1 - FreePercentage(info.FreeBytes, info.Capacity)));
```

Comment:
I am not sure if these _really_ add value... It is truly a single division operation.

Reply:
Well I thought about which values to present. If we do not do this it will always round down, now it will return 1 if 0.9% of space is free.

Reply:
I actually meant the entire method. I feel that this operation is _OK_ but not particularly useful as I can do `si.Free / si.Total` in my own code.